themes/src/main/resources/theme/keycloak.v2/login/register.ftl

```diff
@@ -46,5 +45,39 @@
             </div>
 
         </form>
+        <script type="module">
+            import { validatePassword } from "${url.resourcesPath}/js/password-policy.js";
+
+            const template = `
+            <div class="pf-v5-c-form__helper-text" aria-live="polite">
```

Comment:
Should this not also be using the CSS classes provided in the properties? If you want to build a DOM structure another option is to use a [`<template>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template). This element can be inserted into the DOM, but is not rendered itself, and can be used to build up a new element to insert into the DOM.

This would also remove the need to do a text replace on `{errors}` as you could instead use a regular `querySelector()` to find the correct element and update it.